src/vl-mapactions-custom-map.js

```diff
@@ -84,7 +97,7 @@ export class VlCustomMap extends VlMapWithActions {
     this.overviewMapControl.element.addEventListener('click', () => toggleBaseLayer(), false);
 
     if (options.view) {
-      options.controls.push(this.overviewMapControl);
+      this.addControl(this.overviewMapControl);
```

Comment:
Als ik naar de vorige code kijk, lijkt de overviewMapControl niet altijd te bestaan. Het uitvoeren van `addControl` met null als argument zorgt voor problemen.

Reply:
Bij het volledig bekijken van de code is dit in orde, de `overviewMapControl` is op dat niveau altijd gedefinieerd.